src/components/auth/ForgotPassword.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import FormInput from "../common/FormInput";
+import { useNavigate } from "react-router-dom";
+import Modal from 'react-modal';
+
+Modal.setAppElement('#root'); // Set the app element for accessibility
+
+const ForgotPassword = () => {
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [formErrors, setFormErrors] = useState({});
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (!email) {
+      setFormErrors({
+        ...formErrors,
+        email: "Email is required",
+      });
+      return;
+    }
+
+    try {
+      const response = await axios.post('http://localhost:8000/api/v1/forgot-password', {
+        email: email
+      }, {
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      });
+      setMessage(''); // Clear the message
+      setIsModalOpen(true); // Open the modal
+    } catch (error) {
+      console.error('Error:', error);
+      if (error.response && error.response.status === 401) {
+        setMessage('Invalid credentials');
```

Comment:
Since this is expecting only an email. It is fine to say wrong email. Credentials is also fine but it can give the impression that it's expecting something else to the less savoy internet user.